esmvaltool/diag_scripts/weighting/weighted_temperature_map.py

```diff
@@ -61,9 +61,10 @@ def set_antimeridian(dataarray, to: str):
 def mapplot(dataarray, cfg, title_pattern, filename_part, ancestors,
             **colormesh_args):
     """Visualize weighted temperature."""
-    period = '{start_year}-{end_year}'.format(**read_metadata(cfg)['tas'][0])
-    if 'tas_reference' in read_metadata(cfg).keys():
-        meta = read_metadata(cfg)['tas_reference']
+    period = '{start_year}-{end_year}'.format(
+        **read_metadata(cfg)['tas_CLIM_future'][0])
+    if 'tas_CLIM_reference' in read_metadata(cfg).keys():
+        meta = read_metadata(cfg)['tas_CLIM_reference']
         period = 'change: {} minus {start_year}-{end_year}'.format(
             period, **meta[0])
```

Comment:
If we define `metadata` above, we can simplify this too:

```suggestion
    if 'tas_CLIM_reference' in metadata:
        metadata_reference = metadata['tas_CLIM_reference']
        start_year_ref = metadata_reference[0]['start_year']
        end_year_ref = metadata_reference[0]['end_year']
        
        period = f'change: {period} minus {start_year_ref}-{end_year_ref}'
```